### PR TITLE
 Version and other Updates for ARM template

### PR DESCRIPTION
**Pull request by priyansh-sumo**

### CODE REVIEW

File: EventHubs/tests/azure_metrics.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from azure.servicebus import ServiceBusService
+from requests import Session
+import json
+from datetime import datetime
+import time
```
Comment (himanshu219) [inline EventHubs/tests/azure_metrics.py:5]: delete this file we already have test_eventhub_metrics.py create a function insert_mock_metrics_in_EventHub and in check error logs we can use this code https://learn.microsoft.com/en-us/python/api/overview/azure/monitor-query-readme?view=azure-python#resource-logs-query
you may also have to update the test dependencies in requirements.txt

File: EventHubs/src/azuredeploy_metrics_with_appinsights.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```
Comment (himanshu219) [inline EventHubs/src/azuredeploy_metrics_with_appinsights.json:2]: lets delete this file and copy paste it's code to azuredeploy_metrics.json file we will have single file.

**Review by himanshu219 — CHANGES_REQUESTED** [review]
minor comments

File: EventHubs/tests/azure_metrics.py (hunk 1 above)
Comment (priyansh-sumo) [reply EventHubs/tests/azure_metrics.py]: Deleted

**Pull request merged**